includes/discount-functions.php

```diff
@@ -1584,7 +1584,7 @@ function edd_discount_status_cleanup() {
 			),
 			array(
 				'key'     => '_edd_discount_expiration',
-				'value'   => current_time( 'mysql' ),
+				'value'   => date( 'm/d/Y H:i:s' ),
```

Comment:
please pass `current_time( 'timestamp' )` in as the second parameter to ensure the timezone is correct.